doc/source/contributing_code.rst

```diff
@@ -69,32 +69,38 @@ You can do this either with :ref:`conda <dev-with-conda>` or :ref:`pip <dev-with
 
 .. _dev-with-conda:
 
-With conda
-----------
+With conda-forge packages
+-------------------------
+
+We will use pre-compiled packages from the `conda-forge <https://conda-forge.org/>`_ repository.
+
+The program to manage these packages is called :program:`conda` although for the following we recommend using the faster one called :program:`mamba`. We will use ``mamba`` for the examples but you can equally use ``conda`` instead.
 
-Install either `Anaconda <https://www.anaconda.com/download/>`_
+* For ``mamba``, follow the `mamba installation instructions <https://mamba.readthedocs.io/en/latest/installation/mamba-installation.html>`_ or the `miniforge installation instructions <https://github.com/conda-forge/miniforge?tab=readme-ov-file#install>`_.
+* For ``conda`` install either the  `Anaconda <https://www.anaconda.com/download/>`_ distribution or
 or `miniconda <https://conda.io/miniconda.html>`_.
+
 Make sure your conda is up to date:
 
     .. code-block:: bash
 
-        conda update conda
+        mamba update mamba
 
-Create a new environment with ``conda create``. This will allow you to change code in
+Create a new environment with ``mamba create``. This will allow you to change code in
 an isolated environment without touching your base Python installation, and without
 touching existing environments that may have stable versions of MDAnalysis. :
 
     .. code-block:: bash
 
-        conda create --name mdanalysis-dev "python>=3.9"
+        mamba create --name mdanalysis-dev "python>=3.10"
```

Comment:
Should we recommend the latest supported Python here instead? We will have three version changes before the docs going out of date. ;)

Reply:
Ok. Can I ask you to try out the installation instructions below for the version you want to put here?

Reply:
I'm sorry but I'm AFK until Friday morning (UTC+2). Please discard my comment if this needs to be merged quickly, it was just a "nice to have" suggestion.